data/json/hobbies.json

```
@@ -397,21 +598,51 @@
     "skills": [ { "level": 2, "name": "melee" }, { "level": 3, "name": "unarmed" }, { "level": 3, "name": "dodge" } ],
     "traits": [ "MARTIAL_ARTS3" ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "blackbelt_hobby",
+    "name": "Black Belt",
+    "description": "You've been doing martial arts for years and have earned the coveted black belt in a martial art of your choice.",
+    "points": 10,
+    "skills": [
+      { "level": 5, "name": "melee" },
+      { "level": 5, "name": "unarmed" },
+      { "level": 5, "name": "dodge" },
+      { "level": 3, "name": "swimming" }
```

```suggestion
      { "level": 5, "name": "dodge" }
```
Probably forgot to remove swimming after copy-paste